misc/musl-static/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM clux/muslrust:1.61.0
 
 ARG ARCH=x86_64
 
+RUN rustup component add clippy && rustup component add rustfmt && \
+  rustup target add $ARCH-unknown-linux-musl
```

Comment:
`$ARCH` is put in `RUN` instruction, does this work with `aarch64` or other archs?